session/SessionHandler.php

```diff
@@ -95,6 +95,40 @@ public function read($session_id);
 	public function write($session_id, $session_data);
 }
 
+/**
+ * <b>SessionUpdateTimestampHandlerInterface</b> is an interface which
```

Comment:
did you use any resource to write this descrption?

Reply:
Actually I wrote it by myself but tried to make it similar to SessionHandlerInterface.

One "resource" has been a RFC which describes the behavior quite good but is not directly related to the implementation. Also I am using it by myself.

https://wiki.php.net/rfc/session-read_only-lazy_write

I had not much time lately so couldn't work on official documentation.